### ISSUE TITLE
Running CPU 2.4.1 on Ubuntu, what is the correct process to add the Tensorflow C libraries?

### ISSUE THREAD

**Issue by LukeBolly** [SciSharp/TensorFlow.NET#811]
I've got this working on my local windows machine, however our build server which is running Ubuntu fails to load the library

```
System.DllNotFoundException : Unable to load shared library 'tensorflow' or one of its dependencies. 
In order to help diagnose loading problems, consider setting the LD_DEBUG environment variable:
 libtensorflow: cannot open shared object file: No such file or directory
```

I've got SciSharp.Tensorflow.Redist 2.4.1 installed, and the .dll's + .so's are being copied to netcoreapp3.1\runtimes as expected on my local machine.

I've tried adding a build step to copy the .so files to the run directory
```
  <Target Name="copylibtf" AfterTargets="AfterBuild">
    <Copy SourceFiles="$(outDir)runtimes\linux-x64\native\libtensorflow.so" DestinationFolder="$(OutDir)" />
  </Target>
  <Target Name="copylibtfframework" AfterTargets="AfterBuild">
    <Copy SourceFiles="$(outDir)runtimes\linux-x64\native\libtensorflow_framework.so.2"
          DestinationFiles="$(OutDir)libtensorflow_framework.so" />
  </Target>
```
I confirmed the presence of the files in /netcoreapp3.1 after building, but still the same error.

What should I do here?

**Comment by Oceania2018** [comment]
Can you try to use this version? https://storage.googleapis.com/tensorflow/libtensorflow/libtensorflow-cpu-linux-x86_64-2.5.0.tar.gz

**Comment by LukeBolly** [comment]
Is this file in any of the Nugets? GitHub doesn't allow committing files bigger than 100Mb, so I need to pull it remotely. Can I confirm that release/netcoreapp3.1/ is the correct directory to put the libtensorflow.so and libtensorflow_framework.so?

**Comment by LukeBolly** [comment]
I've just tested with the .so's from that version, using this to copy
```
            wget https://storage.googleapis.com/tensorflow/libtensorflow/libtensorflow-cpu-linux-x86_64-2.5.0.tar.gz
            tar -xzvf libtensorflow-cpu-linux-x86_64-2.5.0.tar.gz

            cp "./lib/libtensorflow.so.2.5.0" "test/MyProj/bin/Release/netcoreapp3.1/"
            cp "./lib/libtensorflow_framework.so.2.5.0" "test/MyProj/bin/Release/netcoreapp3.1/"
            mv "test/Alerte.Holter.Clustering.Test/bin/Release/netcoreapp3.1/libtensorflow.so.2.5.0" "test/MyProj/bin/Release/netcoreapp3.1/libtensorflow.so"
            mv "test/MyProj/bin/Release/netcoreapp3.1/libtensorflow_framework.so.2.5.0" "test/MyProj/bin/Release/netcoreapp3.1/libtensorflow_framework.so"
```

I've verified the files are in the run directory
![image](https://user-images.githubusercontent.com/24449147/118294928-d6ed2480-b50d-11eb-98ce-9e837a380d90.png)

and I still get the error:
```
System.AggregateException : 
One or more errors occurred. (Unable to load shared library 'tensorflow' or one of its dependencies. 
In order to help diagnose loading problems, consider setting the LD_DEBUG environment variable: 
libtensorflow: cannot open shared object file: No such file or directory)
---- System.DllNotFoundException : Unable to load shared library 'tensorflow' or one of its dependencies. 
In order to help diagnose loading problems, consider setting the LD_DEBUG environment variable:
libtensorflow: cannot open shared object file: No such file or directory
```

**Comment by LukeBolly** [comment]
After finding this, https://github.com/migueldeicaza/TensorFlowSharp/issues/476 I tried copying all the files even though they appear to be the same, and it now works. It seems like there are missing files for the Linux in the Redist Nuget.

**Comment by Oceania2018** [comment]
Released [tf 2.5.0 nuget](https://www.nuget.org/packages/SciSharp.TensorFlow.Redist/2.5.0).
